portal/config/eproms/Organization.json

```diff
@@ -1318,6 +1318,34 @@
       },
       "resourceType": "Organization"
     },
+    {
+      "extension": [
+        {
+          "timezone": "America/New_York",
```

Comment:
I think this should technically be [`America/Toronto`](https://time.is/Hamilton), but the offset should be the same